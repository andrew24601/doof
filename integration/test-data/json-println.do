class foo {
    x: int;
    y: int;
}

function main(): void {
    const y:foo = {x: 10, y: 12};
    const x = 12;
    println("Hello world!");
    println(y);
    println("How are you? " + y);
}
