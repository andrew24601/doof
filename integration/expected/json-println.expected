Hello world!
{"x":10,"y":12}
How are you? {"x":10,"y":12}
